htdocs/comm/card.php

```diff
@@ -228,7 +228,7 @@
 	// set prospect level
 	if ($action == 'setprospectlevel' && $user->hasRight('societe', 'creer')) {
 		$object->fetch($id);
-		$object->fk_prospectlevel = GETPOST('prospect_level_id', 'alpha');
+		$object->fk_prospectlevel = GETPOSTINT('prospect_level_id');
```

Comment:
fk_prospectlevel is a "false friend". It real type is string. I have fixed that in develop.

Reply:
hum, tricky one.
fixed now.